dd-sdk-android/src/main/kotlin/com/datadog/android/DatadogSite.kt

```diff
@@ -31,51 +33,35 @@ enum class DatadogSite(val siteName: String) {
     US5("us5"),
 
     /**
-     *  The US1_FED site (FedRAMP compatible): [app.ddog-gov.com](https://app.ddog-gov.com).
+     *  The EU1 site: [app.datadoghq.eu](https://app.datadoghq.eu).
      */
-    US1_FED("us1_fed"),
+    EU1("eu1", "browser-intake-datadoghq.eu"),
 
     /**
-     *  The EU1 site: [app.datadoghq.eu](https://app.datadoghq.eu).
+     *  The AP1 site: [ap1.datadoghq.com](https://ap1.datadoghq.com).
      */
-    EU1("eu1");
+    AP1("ap1"),
 
     /**
-     * Returns the endpoint to use to upload Logs to this site.
+     *  The US1_FED site (FedRAMP compatible): [app.ddog-gov.com](https://app.ddog-gov.com).
      */
-    fun logsEndpoint(): String {
-        return when (this) {
-            US1 -> DatadogEndpoint.LOGS_US1
-            US3 -> DatadogEndpoint.LOGS_US3
-            US5 -> DatadogEndpoint.LOGS_US5
-            US1_FED -> DatadogEndpoint.LOGS_US1_FED
-            EU1 -> DatadogEndpoint.LOGS_EU1
-        }
-    }
+    US1_FED("us1_fed", "browser-intake-ddog-gov.com"),
 
     /**
-     * Returns the endpoint to use to upload Spans to this site.
+     *  The STAGING site (internal usage only): [app.datad0g.com](https://app.datad0g.com).
```

Comment:
Is it useful to expose it in the public API? Isn't using custom endpoint URL API enough?

Reply:
It is already exposed in the browser SDK, so it's not really a secret, and it can make our life during tests easier